### ISSUE TITLE
Wrong name of the global symbol in LinkerScript part

### ISSUE THREAD

**Issue by oliwkowemango** [umanovskis/baremetal-arm#6]
>  This script tells the linker that the program's entry point is at the global symbol `_Entry`, which we export from `startup.s`.

Should it be a `_Reset` instead of `_Entry`?

**Comment by csukuangfj** [comment]
@oliwkowemango 

yes, you're right. You can see that it actually uses `_Reset` in the linker script:

https://github.com/umanovskis/baremetal-arm/blob/4ce7973889bbc1d64299e4a87634f471137e3761/src/04_cenv/linkscript.ld#L3

https://github.com/umanovskis/baremetal-arm/blob/4ce7973889bbc1d64299e4a87634f471137e3761/src/04_cenv/linkscript.ld#L13-L15

https://github.com/umanovskis/baremetal-arm/blob/4ce7973889bbc1d64299e4a87634f471137e3761/src/04_cenv/startup.s#L6-L10
